config/dc2gen3.yaml

```diff
@@ -0,0 +1,17 @@
+gen2_root: /data/des90.a/data/neilsen/dc2gen3_tests/2021-01-28/gen2
+gen3_root: /data/des90.a/data/neilsen/dc2gen3_tests/2021-01-28/gen3
+instrument: lsst.obs.lsst.LsstCamImSim
+delete_old_gen3: True
+create_new_gen3: True
+convertRepo_config: convertrepo.pexconfig
+butler_seed: butler_seed.yaml
+reruns:
+    - path: rerun/run2.2i-calexp-v1
+      runName: run2.2i-calexp-v1
+      chainName: null
+      parents: []
```

Comment:
FYI, we will need changes to these parameters to match the conventions in [DMTN-167](https://dmtn-167.lsst.io/), but we'll probably need some slight changes to `ConvertRepoTask` and/or `obs_lsst` as well, and it'll be best to consider all of the reruns to be converted together.  I don't think that's a job for this ticket, but it's definitely something we should do before starting the full conversion.

Reply:
The runs here reflect the organization of the runs as they exist in the DC2 gen2 repo. If runs need to be named and organized differently, the reorganization will need to be done either as an additional patch on the DC2 gen2 repo (which I agree would be an additional ticket), or the `ConvertRepoTask` will need to do the reorganization as part of the conversion to gen3.